lib/wrapper.rb

```diff
@@ -0,0 +1,29 @@
+module Wrapper 
+
+	# Wrapper module server's as a wrapper to 
+	# add output to the log file. 
+	# The method log take's in 2 parameters.
+	# A string passed in as level and a hash
+	# passed in as message. Please look at 
+	# examples from the user.rb model. 
+	
+	def self.log(level,message)
+
+	severity_levels = 
+	    {
+		'DEBUG' => 0,
```

Comment:
Let's use the set from https://github.com/NYPL/engineering-general/blob/master/standards/logging.md .

Reply:
Look at line 25. We use a method called add. Add is a built in Ruby method. I'm not sure if it will work properly if you use the engineering-general standard numbers. Does that make sense? 

Read the first 2 paragraphs of the following link: https://guides.rubyonrails.org/debugging_rails_applications.html#log-levels